### DISCUSSION TITLE
桌面版只能登陆一台电脑？在另一台电脑一直报错：没有record

### DISCUSSION THREAD

**Discussion by AlvinLXP** [AlistGo/alist#6005] [Q&A]
![image](https://github.com/alist-org/alist/assets/24516032/d0833464-8c6f-4834-841b-48a2f814e10f)

**Comment by xhofe** [comment]
新电脑打开的软件代表另一个实例，如果想访问另外电脑中的alist，请使用搭建电脑的ip访问。

**Reply by AlvinLXP** [reply]
@xhofe 设置过了还是不行，可以登录三台电脑如何操作；或者是否可以这台电脑卸载，链接删除后换电脑可以用；
